src/output.rs

```diff
@@ -0,0 +1,91 @@
+use std::fmt;
+use std::process::Output;
+
+use difference::Changeset;
+
+use errors::*;
+use diff;
+
+#[derive(Debug, Clone)]
+pub struct OutputAssertion<T> {
+    pub expect: String,
+    pub fuzzy: bool,
+    pub kind: T,
+}
+
+impl<T: OutputType> OutputAssertion<T> {
+    fn matches_fuzzy(&self, got: &str) -> Result<()> {
+        if !got.contains(&self.expect) {
+            bail!(ErrorKind::OutputMismatch(
+                self.kind.to_string(),
+                vec!["Foo".to_string()],
```

Comment:
Placeholder for now. I think we can use error_chain to add the usual "Assert CLI command {cmd} failed:" prefix at the call site.